lib/config.js

```diff
@@ -14,6 +14,13 @@ const conf = convict({
       default: 1
     }
   },
+  authServer: {
+    url: {
+      doc: 'URL of fxa-auth-server',
+      format: 'url',
```

Comment:
I think we are trying to have ENV variables for these. seems like this one is missing a way to set this setting via an ENV var.

Ref: https://github.com/mozilla/fxa-auth-server/blob/master/config/index.js#L85-L102